apps/web/components/eventtype/EventTeamTab.tsx

```diff
@@ -189,14 +261,19 @@ const Hosts = ({
                 />*/}
             </>
           ),
+          MANAGED: <></>,
```

Comment:
Necessary to avoid an error in the following line expecting to have all scheduling types covered.